.github/workflows/issue-label-notifier.yaml

```diff
@@ -0,0 +1,17 @@
+name: Notify users based on issue labels
+
+on:
+  issues:
+    types: [labeled]
+  pull_request:
+    types: [labeled]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jenschelkopf/issue-label-notification-action@1.3
+        with:
+          recipients: |
+            design-review=@Katherine-Osos
+          message: 'cc/ {recipients} — adding you to this **{label}** issue!'
```

Comment:
Looks like this change adds both an end quote and a newline - newline seems fine, but I think the extra quote mark will cause issues. @zandercymatics  quick check on that? Otherwise I can just add the newline.

Reply:
Yeah remove the extra end quote, but the new line makes sense